### PR TITLE
User-Interface/local-toggle-styling

### PR DESCRIPTION
**Pull request by BrianWhitneyAI**
## Description
The purpose of this PR is to update the styling of the "toggle local uploads feature" to reflect UX design in #119 

### SPEC
image

### CODE REVIEW

**Comment by BrianWhitneyAI** [comment]
Current Appearance:
Screenshot 2024-10-29 at 12 32 50 PM

**Review by TravisKroeker — APPROVED** [review]

File: src/renderer/containers/UploadWithTemplatePage/PageFooter/index.tsx [hunk 1]
```diff
@@ -79,12 +78,16 @@ export default function PageFooter(props: Props) {
...
           checked={ShouldBeInLocal}
-          onChange={onCheckboxChange}
-        >
-          Store Locally
-        </Checkbox>
+          onChange={onSwitchChange}
+          checkedChildren="On"
+          unCheckedChildren="Off"
+        />
+        <span style={{ marginLeft: 8 }}>Only store on cloud</span>
```
Comment (SeanLeRoy) [inline src/renderer/containers/UploadWithTemplatePage/PageFooter/index.tsx:87]: Generally this sort of css styling should go into the css file paired with the component (like on line 80 where it is assigned a className)

**Review by SeanLeRoy — APPROVED** [review]

File: src/renderer/containers/UploadWithTemplatePage/PageFooter/index.tsx [hunk 2]
```diff
@@ -47,9 +46,9 @@ export default function PageFooter(props: Props) {
     }
   }
 
-  // Handler to update ShouldBeInLocal state when the checkbox is toggled
-  const onCheckboxChange = (e: CheckboxChangeEvent) => {
-    dispatch(setShouldBeInLocal(e.target.checked));
+  // Handler to update ShouldBeInLocal state when the switch is toggled
+  const onSwitchChange = (checked: boolean) => {
+    dispatch(setShouldBeInLocal(checked));
```
Comment (saeliddp) [inline src/renderer/containers/UploadWithTemplatePage/PageFooter/index.tsx:51]: I think `shouldBeInLocal` should be equivalent to`!checked`--if the switch is 'on' (`checked=true`, I assume), it's a cloud-only upload, so `shouldBeInLocal` should be `false`

**Review by saeliddp — CHANGES_REQUESTED** [review]

File: src/renderer/containers/UploadWithTemplatePage/PageFooter/index.tsx (hunk 2 above)
Comment (BrianWhitneyAI) [reply src/renderer/containers/UploadWithTemplatePage/PageFooter/index.tsx]: resolved in [8e6f957](https://github.com/aics-int/aics-file-upload-app/pull/120/commits/8e6f957f0e6697814efd10e9f804a8d71ba4c5dc)

**Comment by BrianWhitneyAI** [comment]
Just ran a test with the switch checked and everything looks good. Heres the response body from swagger

```
{
  "fileId": "1438a285e8a84b34922b1e2d7dcb976d",
  "name": "test14.csv",
  "type": "csv",
  "size": 341,
  "md5": "5688a2a31e2c381e3f9677a73942117f",
  "cloudPath": "staging.files.allencell.org/76d/cb9/d7d/1e2/22b/349/84b/e8a/285/38a/14/test14.csv",
  "localPath": null,
  "created": 1730764038297,
  "file_id": "1438a285e8a84b34922b1e2d7dcb976d",
  "cloud_path": "staging.files.allencell.org/76d/cb9/d7d/1e2/22b/349/84b/e8a/285/38a/14/test14.csv",
  "local_path": null
}
```

**Review by saeliddp — APPROVED** [review]

**Pull request merged**